README.md

```diff
@@ -50,27 +53,28 @@ $ $GOPATH/bin/terraform-provider-newrelic
 ...
 ```
 
-In order to test the provider, you can simply run `make test`.
+#### Testing
+In order to test the provider, run `make test`. This will run the unit test suite.
 
 ```sh
 $ make test
 ```
 
 In order to run the full suite of Acceptance tests, run `make testacc`.
 
-*Note:* Acceptance tests create real resources, and often cost money to run.
+*Note:* Acceptance tests *create real resources*, and often cost money to run. The environment variable `NEWRELIC_API_KEY` must also be set with your key for acceptance tests to work properly.
```

Comment:
`NEWRELIC_LICENSE_KEY` is also currently needed for acc testing.

Reply:
Ah, right. Thanks.